pages/[[...app]].tsx

```diff
@@ -0,0 +1,99 @@
+// import '@reach/dialog/styles.css'
+import 'inter-ui'
+import '../src/polyfills'
+
+// import '../src/components/analytics'
+import { ApolloProvider } from '@apollo/client'
+import React, { useEffect, useState } from 'react'
+import { StrictMode } from 'react'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { Provider } from 'react-redux'
+import { HashRouter } from 'react-router-dom'
+
+import Web3Provider from '../src/components/Web3Provider'
+import { FeatureFlagsProvider } from '../src/featureFlags'
+import { apolloClient } from '../src/graphql/data/apollo'
+import { LanguageProvider } from '../src/i18n'
+import { BlockNumberProvider } from '../src/lib/hooks/useBlockNumber'
+import { MulticallUpdater } from '../src/lib/state/multicall'
+import App from '../src/pages/App'
```

Comment:
This file is a temporary copy of our main entry point. Tomorrow, I will add some exports to the original file and as a result, this will become much thinner, leveraging `_app.tsx` structure too.